### ISSUE TITLE
Should only actually check if button is pressed

### ISSUE THREAD

**Issue by marcusreid** [hsbakshi/reddit-check#4]
This seems way, way too aggressive. It hits reddit for every page you visit. Not to mention the privacy concerns. It seems like it should default to only doing a lookup when a button is pressed.

**Comment by mlebkowski** [comment]
I’ve created a fork that only checks the url when the icon is clicked: https://github.com/mlebkowski/reddit-check

**Comment by Ajedi32** [comment]
Yeah, this just occurred to me as well. @mlebkowski is that fork something you're planning to maintain? I'm thinking of making my own fork and publishing to the Chrome store, but there are a few changes I want to make first. (Bringing that fork up to speed with the latest changes to upstream, and removing the tabs permission.)

**Comment by mlebkowski** [comment]
@Ajedi32, I am not commited to this fork, browser extensions are not something I do. Feel free to use my code if you think it’s useful.

I would gladly switch from my version to a maintained one if you ever make it.

**Comment by mmortal03** [comment]
@mlebkowski , @Ajedi32 are you guys still using this extension, or have you found something better in the last few years?

**Comment by mlebkowski** [comment]
@mmortal03, I actually use my version (sporadically), never looked back

**Comment by Ajedi32** [comment]
@mmortal03 Not using this extension anymore, but it looks like there's a slightly more active (though still outdated) fork at https://github.com/ronnyli/thredd. 357 commits ahead

**Comment by mmortal03** [comment]
> @mmortal03 Not using this extension anymore, but it looks like there's a slightly more active (though still outdated) fork at https://github.com/ronnyli/thredd. 357 commits ahead

Wow, that one would be great if it didn't insert an extra forward slash in the direct URLs (e.g. http://www.reddit.com//r/politics/), which breaks RES. Someone reported the bug, but it hasn't been fixed: https://github.com/ronnyli/thredd/issues/114

Maybe I'll give mlebkowski's version a shot.
